app/src/main/java/xyz/whereuat/whereuat/utils/PendingRequestUtils.java

```diff
@@ -0,0 +1,38 @@
+package xyz.whereuat.whereuat.utils;
+
+import android.content.ContentValues;
+import android.content.Context;
+
+import xyz.whereuat.whereuat.db.command.DeleteCommand;
+import xyz.whereuat.whereuat.db.command.InsertCommand;
+import xyz.whereuat.whereuat.db.command.QueryCommand;
+import xyz.whereuat.whereuat.db.entry.PendingRequestEntry;
+
+/**
+ * Utility functions for dealing with pending requests.
+ */
+public class PendingRequestUtils {
+    public static InsertCommand buildInsertCommand(Context context, String name, String phone) {
+        ContentValues values = new ContentValues();
+        values.put(PendingRequestEntry.COLUMN_NAME, name);
+        values.put(PendingRequestEntry.COLUMN_PHONE, phone);
+        return new InsertCommand(context, PendingRequestEntry.TABLE_NAME, null, values);
+    }
+
+    public static QueryCommand buildSelectAllCommand(Context context) {
+        return new QueryCommand(context, PendingRequestEntry.TABLE_NAME, false,
+                PendingRequestEntry.COLUMNS, null, null, null, null, null, null);
```

Comment:
I completely forgot about this for the previous PR, but an argument of `null` automatically returns all columns -- no need for the `.COLUMNS` member here